modules/drivers/oracle/test/metabase/driver/oracle_test.clj

```diff
@@ -12,6 +13,7 @@
             [metabase.models.table :refer [Table]]
             [metabase.query-processor :as qp]
             [metabase.query-processor-test :as qp.test]
+            [metabase.query-processor-test.order-by-test] ; used for one SSL connectivity test
```

Comment:
You can just alias this as `qp-test.order-by-test` or something like that, it doesn't matter too much

Reply:
Done.